packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/events-queue.ts

```diff
@@ -0,0 +1,184 @@
+import * as ecs from '@aws-cdk/aws-ecs';
+import * as sns from '@aws-cdk/aws-sns';
+import * as subscription from '@aws-cdk/aws-sns-subscriptions';
+import * as sqs from '@aws-cdk/aws-sqs';
+import * as cdk from '@aws-cdk/core';
+import { Service } from '../service';
+import { Container } from './container';
+import { ContainerMutatingHook, ServiceExtension } from './extension-interfaces';
+
+// Keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * The settings for the EventsQueue extension.
+ */
+export interface EventsQueueProps {
+  /**
+   * The list of topic subscriptions for this service.
+   *
+   * @default none
+   */
+  readonly topicSubscriptions?: TopicSubscriptionProps[];
```

Comment:
I wonder if we can make this even more generic for the future so that it's not only topic subscriptions that we accept but also eventbus subscriptions.

We could create a `Susbcribable` interface that a `TopicSubscription` class implements, here is a bad skeleton 😛 : https://gist.github.com/efekarakus/78567b5f4aff33434963dc42080be541